src/support/utf8.c

```diff
@@ -615,71 +615,63 @@ size_t u8_printf(const char *fmt, ...)
     return cnt;
 }
 
-/* based on the valid_utf8 routine from the PCRE library by Philip Hazel
+/* Rewritten completely, original code not based on anything else
 
    length is in bytes, since without knowing whether the string is valid
    it's hard to know how many characters there are! */
-int u8_isvalid(const char *str, size_t length)
+int u8_isvalid(const char *iStr, size_t iLength)
```

Comment:
@tknopp to be fair, I think @ScottPJones has gotten the message about opening concrete, independent, small issues by now. See #11123 for an issue about doc comments. On doc comments and variable naming I think it might be best to just start proposing small PR's to solely add doc comments or replace variable names with clearer, more consistent versions. They may stand out as not matching the surrounding code at first, but if a consistent style starts getting gradually applied to more of the C code then I think it could be fair to start holding newly added code to a standard (which we would add to CONTRIBUTING.md).